source_gen/test/generator_for_annotation_test.dart

```diff
@@ -161,7 +161,18 @@ $dartFormatWidth
     final builder = LibraryBuilder(
       _StubGenerator<Deprecated>(
         'Deprecated',
-        directiveBehavior: (element) => '// ${element.runtimeType}',
+        directiveBehavior: (element) {
+          switch (element) {
```

Comment:
refactor into switch expression – is nice!

Reply:
Done.